source/puppet/4.4/reference/static_catalogs.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "Static Catalogs"
+canonical: "/puppet/latest/reference/static_catalogs.html"
+---
+
+[catalogs]: ./subsystem_catalog_compilation.html
+[catalog endpoint]: ./http_api/http_catalog.html
+[`static_file_content`]: /puppetserver/latest/puppet-api/v3/static_file_content.html
+[resource_declaration]: ./lang_resources.html
+[file resources]: ./types/file.html
+[puppet catalog]: ./man/catalog.html
+[environment]: ./environments.html
+[facts]: ./lang_facts_and_builtin_vars.html
+[exported resources]: ./lang_exported.html
+[main manifest]: ./dirs_manifest.html
+[modules]: ./modules_fundamentals.html
+[resources]: ./lang_resources.html
+[variables]: ./lang_variables.html
+[classes]: ./lang_classes.html
+[modulepath]: ./dirs_modulepath.html
+[`puppet.conf`]: ./config_file_main.html
+[`environment.conf`]: ./config_file_environment.html
+
+[Puppet Server]: /puppetserver/latest/
+[`puppetserver.conf`]: /puppetserver/latest/config_file_puppetserver.html
+[Application Orchestration]: /pe/latest/app_orchestration_overview.html
+[file sync]: /pe/latest/cmgmt_filesync.html
+[Code Manager]: /pe/latest/code_mgr.html
+[`code_content`]: /puppetserver/latest/
+
+Puppet 4.4 and Puppet Server 2.3 introduce a new feature for [catalog compilation][catalogs]: **static catalogs**.
+
+> #### What's a catalog?
+>
+> A catalog is a document that describes the [desired state for each resource][resource_declaration] that Puppet should manage on a node. A Puppet master typically compiles a catalog from manifests of Puppet code. For more information about catalogs and compilation, see [our step-by-step overview of the catalog compilation process][catalogs]. For details about retrieving and reviewing catalogs, see [the `puppet catalog` man page][puppet catalog] and [the `catalog` API endpoint documentation][catalog endpoint].
+
+### What's a static catalog?
+
+Unlike a standard Puppet catalog, a static catalog includes metadata that identifies the desired state of any [file resources][] on a node that have `source` attributes pointing to `puppet:///` locations. This metadata can refer to a specific version of the file, rather than the latest version, and confirm that the agent is applying the appropriate version of the file resource for the catalog. Also, because much of the metadata is provided in the catalog, Puppet agents make fewer requests to the master.
+
+### Why use static catalogs?
+
+When a Puppet master produces a normal catalog, the result doesn't specify the a version of file resources. When the agent applies the catalog, it always retrieves the latest version of that file resource.
```

Comment:
The end of this paragraph could give the impression that the agent always fetches a file from the master, even if it's already fetched it in the past and has it on disk.  I might adjust the wording to avoid that misunderstanding.